src/common/dav/ulog.h

```diff
@@ -74,12 +75,33 @@ VEC(ulog_next, struct ulog *);
 typedef uint64_t ulog_operation_type;
 
 #define ULOG_OPERATION_SET		(0b000ULL << 61ULL)
+#ifdef	WAL_SUPPORTS_AND_OR_OPS
 #define ULOG_OPERATION_AND		(0b001ULL << 61ULL)
 #define ULOG_OPERATION_OR		(0b010ULL << 61ULL)
+#else
+#define ULOG_OPERATION_CLR_BITS		(0b001ULL << 61ULL)
+#define ULOG_OPERATION_SET_BITS		(0b010ULL << 61ULL)
```

Comment:
(style)  Avoid gcc v4.3+ binary constant extension: <0b010ULL>

Reply:
Please make an exemption for gcc v4.3+ binary constant extension.